com.sigil_ebook.Sigil.yml

```diff
@@ -0,0 +1,139 @@
+app-id: com.sigil_ebook.Sigil
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: 5.15-21.08
+rename-desktop-file: sigil.desktop
+rename-icon: sigil
+command: sigil
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --filesystem=xdg-documents
```

Comment:
I can imagine it should be `home` as you cannot really know where someone stores their books.

Reply:
since it's an editor, it's to save them. isn't it?